pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -126,15 +126,15 @@ func (r *ReconcileCertificateRequest) Reconcile(request reconcile.Request) (reco
 	if len(os.Getenv(fedrampEnvVariable)) == 0 {
```

Comment:
Since you're already in this code, can we change this to use os.LookupEnv instead?